subsys/bluetooth/controller/Kconfig

```
@@ -635,6 +635,39 @@ config BT_CTLR_SDC_CS_EVENT_LEN_DEFAULT
 	help
 	  The time set aside for channel sounding events in microseconds.
 
+choice BT_CTLR_SDC_CS_T_PM_LEN_DEFAULT_SELECTION
+prompt "Default channel sounding t_pm length selection"
```

a choice by nature is a selection, especially when seen in `menuconfig`.
```suggestion
prompt "Default channel sounding t_pm length"
```